src/main/java/uk/gov/hmcts/reform/jobscheduler/services/s2s/S2sClient.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.jobscheduler.services.s2s;
+
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+public interface S2sClient {
```

Comment:
Would it be better if we name this class as `S2SClient`?

Reply:
I was going for consistency with https://github.com/hmcts/draft-store/blob/master/src/main/java/uk/gov/hmcts/reform/draftstore/service/s2s/S2sClient.java which I was going to pull out to a library. But after talking to @doncem I'm hoping I can delete all this code first ;)